src/place-detail/ui/place-detail.controller.js

```diff
@@ -165,12 +165,8 @@ function PlaceDetail(
     $state.go('split.footer.dashboard');
   }
 
-  /**
-   * @param {EventCrudJob} job
-   */
-  controller.goToDashboardOnJobCompletion = function(job) {
-    job.task.promise
-      .then(goToDashboard);
+  controller.goToDashboardOnJobCompletion = function() {
```

Comment:
I wonder if we still need this encapsulating method now or if we can straight up call `goToDashboard()` instead

Reply:
Agreed, I think we can remove this and replace it with `goToDashboard` wherever it was used.

Reply:
I agree, I removed the goToDashboard() function and added the function straight to the controller method